generate-certs/main.tf

```diff
@@ -37,8 +36,8 @@ resource "aws_iam_role_policy" "s3_gen_certs" {
         "s3:PutObject"
       ],
       "Resource": [
-        "arn:aws:s3:::${replace(var.s3_root_path,"/(\/)+$/","")}",
```

Comment:
ah, this fun regex helped take care of cases where the root path might have trailing slashes.